src/ankh/cmd/ankh/main.go

```diff
@@ -261,3 +272,43 @@ func check(err error) {
 		os.Exit(1)
 	}
 }
+
+func runScripts(ctx *ankh.ExecutionContext, scripts []struct { Path string }) {
+	for _, script := range scripts {
+		path := script.Path
+		if path == "" {
+			log.Warnf("Missing path in script %s", script)
+			break
+		}
```

Comment:
let's check that the path exists and also that it's executable. these will be the two most common mistakes. we should make sure those checks happen before exiting on dryRun, too.

Reply:
Should be good now!